src/triggers/actor_run_finished.js

```diff
@@ -40,6 +40,14 @@ module.exports = {
                 required: true,
                 dynamic: 'actors.id.name',
             },
+            {
+                label: 'States',
+                helpText: 'Please select the terminal states of the actor run.',
```

Comment:
Actor always with a capital, please check [writing guide](https://www.notion.so/apify/Apify-writing-style-guide-de9fbb99dcd84665b6d3f790fc88b3b6?pvs=4).

Reply:
And the helpText need some love, let's consult this with some copy writer where we will be done with code changes.